common/test/acceptance/pages/lms/progress.py

```diff
@@ -104,3 +119,46 @@ def _section_scores(self, chapter_index, section_index):
 
         # Convert text scores to tuples of (points, max_points)
         return [tuple(map(int, score.split('/'))) for score in text_scores]
+
+
+class PassingInformationTable(PageObject):
+    """
+    PassingInformationTable page object wrapper
+    """
+    url = None
+    TABLE_SELECTOR = '.grade-category-detail-table'
+
+    def __init__(self, browser):
+        super(PassingInformationTable, self).__init__(browser)
+
+    def find_css(self, selector):
+        """
+        Returns a query corresponding to the given CSS selector within the scope
+        of this discussion page
+        """
+        return self.q(css=self.TABLE_SELECTOR + " " + selector)
+
+    def is_browser_on_page(self):
+        """
+        Verify that the browser is on the page and it is not still loading.
+        """
+        return PassingInformationTable.is_present(self)
+
+    @classmethod
+    def is_present(cls, page):
+        """
+        Indicates whether progress page has a PassingInformationTable.
+        """
+        return page.q(css=cls.TABLE_SELECTOR).present
+
+    @property
+    def status(self):
+        """
+        Returns a list of tuples with information about the categories:
+        [('Homework', '50', '25', 'Not pass'), ('Lab', '50', '75', 'Pass')]
+        """
+        rows = self.find_css('tbody tr')
+        # Unfortunetly rows.text returns us information in appropriate fromat.
```

Comment:
Maybe `Unfortunately rows.text returns us information in inappropriate format.`?